spec/support/helpers/admin_login.rb

```diff
@@ -9,6 +9,8 @@ def sign_in_as_super_admin
     end
 
     def sign_in_as(admin_account)
+      Rack::Attack.enabled = false
```

Comment:
ah was rack attack catching those in the tests?

Reply:
yeah rack attack was blocking the sign in page in tests ([/session/sign-in-code](https://github.com/DFE-Digital/npq-registration/blob/main/config/initializers/rack_attack.rb#L15)) max 5 calls per minute.